spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -80,6 +81,11 @@ public Map<String, String> toGravitinoTableProperties(Map<String, String> proper
           HivePropertiesConstants.GRAVITINO_HIVE_TABLE_TYPE,
           HiveTablePropertiesMetadata.TableType.EXTERNAL_TABLE.name());
     }
+
+    if (location != null) {
+      gravitinoTableProperties.put(HivePropertiesConstants.GRAVITINO_HIVE_TABLE_LOCATION, location);
```

Comment:
I prefer to add `location` to `sparkToGravitinoPropertyMap` and new `gravitinoToSparkPropertyMap` to auto convert it.